modules/crafting/server.lua

```diff
@@ -213,6 +213,16 @@ lib.callback.register('ox_inventory:craftItem', function(source, id, index, reci
 							end
 
 							invSlot.count -= 1
+							left:syncSlotsWithClients({
+								{
+									item = invSlot,
+									inventory = left.id
+								},
+								{
+									item = invSlot,
```

Comment:
twice?

Reply:
Actually need to update the slot weight after setting the count, too.

Reply:
wasn't able to find where the weight gets updated